### ISSUE TITLE
[Question] 插件的触发方式是什么？

### ISSUE THREAD

**Issue by Moonlit7** [lobehub/chat-plugin-sdk#25]
### 🧐 问题描述 | Proposed Solution

通过manifest的description关键词来匹配的吗？想了解下这块原理，语言模型如何判断用户意图来调用不同的插件？插件的配置应该如何设置

**Comment by arvinxx** [comment]
> 通过manifest的description关键词来匹配的吗？

对。你可以看下 OpenAI 的[插件开发文档](https://platform.openai.com/docs/plugins/getting-started/plugin-manifest) 了解ChatGPT 插件的详情。我在 LobeChat 做的版本简化了一些

**Comment by Moonlit7** [comment]
目前插件处理完的数据，lobechat似乎没有拼接在上下文里，这样要如何实现基于插件处理结果的后续对话能力？

**Comment by arvinxx** [comment]
> 目前插件处理完的数据，lobechat似乎没有拼接在上下文里，这样要如何实现基于插件处理结果的后续对话能力？

在上下文里的。你点下插件展开来看看就会发现了

**Comment by Moonlit7** [comment]
好的，我再试试，感谢
